index.js

```diff
@@ -98,24 +98,37 @@ io.on('connection', socket => {
   let company = null
   let userPseudonym = null
 
-  const formatMessage = message => `[${company || 'Unknown'} | ${userPseudonym || 'Anonymous'}] ${message}`
+  const formatMessage = message => `[${company.name || 'Unknown'} | ${userPseudonym || 'Anonymous'}] ${message}`
   const info = message => console.log(formatMessage(message))

```

Comment:
You can remove this `info` function now.

Reply:
I left it in as it is still used on lines 111 and 114.